scripts/run_tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+pushd sim
+
+EXIT_CODE=0
+all_features="sig-rsa sig-ecdsa overwrite-only validate-slot0 enc-rsa enc-kw"
+
+if [[ $FEATURES =~ "none" ]]; then
+  cargo test
+  rc=$? && [ $rc -ne 0 ] && EXIT_CODE=$rc
+fi
+
+for feature in $all_features; do
```

Comment:
Although the current test only tests the features one at a time, it does make sense to test some of them together. We probably at least still allow that, whereas this will restrict us to only enabling a single feature at a time.

Reply:
Yes, might be a good idea to be able to provide multiple features per run, will do that!